runtime/src/commonTest/kotlin/pbandk/testpb/test_messages_proto3.kt

```diff
@@ -1707,6 +1707,7 @@ data class TestAllTypesProto3(
                 )
             }
             pbandk.MessageDescriptor(
+                fullName = "protobuf_test_messages.proto3.TestAllTypesProto3",
```

Comment:
Shouldn't this be pbandk.testpb.TestAllTypesProto3?

Reply:
Same here. This is the protobuf message name rather than the Kotlin class name.